lib/susedistribution.pm

```diff
@@ -120,23 +144,21 @@ sub x11_start_program($$$) {
         type_string $program;
     }
     wait_still_screen(1);
-    if ($options->{terminal}) {
-        wait_screen_change { send_key 'alt-t' };
-    }
     save_screenshot;
     send_key 'ret';
-    wait_still_screen unless $options->{no_wait};
-    # lrunner has auto-completion feature, sometimes it causes high load while
```

Comment:
abd why you delete comment ? Looks like it is still valid , no ?

Reply:
I moved it to the documentation above the method which did not exist in before at all.

Reply:
can you point me to a place where you moved ?

Reply:
yes, here: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/3671/files/30ddd6df6bceb37edc92fa8eedc8b1ffeffe3635#diff-a2a433d5914343b617ae52ac0fb40524R111